qa/pull-tester/rpc-tests.py

```diff
@@ -29,6 +29,7 @@
 import tempfile
 import re
 
+sys.path.append("qa/pull-tester/")
```

Comment:
Heh, I run them from qa/pull-tester. I had no idea you could run them from the root...